diff --git a/third_party/llvm-10.0/CMakeLists.txt b/third_party/llvm-10.0/CMakeLists.txt
index c5e1a86ef..6ca42e149 100644
--- a/third_party/llvm-10.0/CMakeLists.txt
+++ b/third_party/llvm-10.0/CMakeLists.txt
@@ -615,6 +615,7 @@ set(LLVM_LIST
     ${LLVM_DIR}/lib/Support/regcomp.c
     ${LLVM_DIR}/lib/Support/regerror.c
     ${LLVM_DIR}/lib/Support/Regex.cpp
+    ${LLVM_DIR}/lib/Support/RWMutex.cpp
     ${LLVM_DIR}/lib/Support/regexec.c
     ${LLVM_DIR}/lib/Support/regfree.c
     ${LLVM_DIR}/lib/Support/regstrlcpy.c
